Formula/libprelude.rb

```diff
@@ -29,7 +30,7 @@ def install
       --without-perl
       --without-swig
       --without-python2
-      --with-python3=#{Formula["python@3.8"].opt_bin/"python3"}
+      --with-python3=python3
```

Comment:
How about
```suggestion
      --with-python3=python3.9
```
or even 
```suggestion
      --with-python3=python#{Formula["python@3.9"].version.major_minor}
```

Reply:
It‘s OK, unless use `#{Formula["python@3.9"].opt_bin}/python#{Formula["python@3.9"].version.major_minor}`.

Reply:
Should we consider this a bug and report it to upstream? From `./configure --help`:
```
...
  --with-python3[=PATH]   Enable support for python3 binding [default=auto]
...
```
(Since it's supposed to work with paths, according to the help text.)